web/src/testgrid-grid-header-row.ts

```diff
@@ -0,0 +1,34 @@
+import { LitElement, html, css } from "lit";
+import { map } from "lit/directives/map.js";
+import { customElement, property } from "lit/decorators.js";
+import { ListHeadersResponse } from './gen/pb/api/v1/data.js';
+
+@customElement('testgrid-grid-header-row')
+export class TestgridGridHeaderRow extends LitElement {
+  static styles = css`
+    :host {
+      text-align: center;
+      font-family: Roboto, Verdana, sans-serif;
+      display: flex;
+      flex-flow: row nowrap;
+      gap: 0px 2px;
+      margin: 2px;
+    }
+  `;
+
+  @property() headers: ListHeadersResponse;
+
+  render() {
+    if (this.headers && this.headers.headers) {
+      return html`
+        <testgrid-grid-row-header></testgrid-grid-row-header>
```

Comment:
so `testgrid-grid-row-header` is just to move this element to the right by the required padding?
in that case, also we could make it DRY by always setting a `testgrid-grid-row-header` and then checking if headers exist

IMO, would also be great to add a TODO to set the padding / margin in this element's instead of using a placeholder element - to place the element correctly

Reply:
Atm it's a placeholder, though I think someone did have an FR that was interesting, suggesting the column header name could go to the left of the column header row to describe what it is. (So date, time, build for the defaults, but the column header name/property we're using for the others. Right now it's a tooltip). That said, even if we do something like that, it shouldn't be the same as a row header 😅